apps/workers/db-check/worker_test.go

```diff
@@ -0,0 +1,201 @@
+package main
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/hydronica/trial"
+	"github.com/jmoiron/sqlx"
+)
+
+func TestGetZeroSums(t *testing.T) {
+	type Row struct {
+		IntCol  int64
+		NumCol  []uint8
+		DateCol time.Time
+	}
+
+	type Input struct {
+		DateType string
+		GroupTS  string
+		Rows     []Row
+	}
+
+	type Expected struct {
+		ZR ZeroRecs
+		MR MissingRecs
+	}
+
+	d := time.Date(2023, 06, 15, 0, 0, 0, 0, time.UTC)
+	dt := make([]time.Time, 24)
+	for i := range dt {
+		dt[i] = d.Add(time.Hour * time.Duration(i))
+	}
```

Comment:
I'd moved this into the test case.

Reply:
Wouldn't it be better to do once, so that all test cases can use it?